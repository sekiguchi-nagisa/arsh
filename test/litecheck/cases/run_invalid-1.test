
# RUN: echo ${34 / "hello"}
# CHECK: 34
