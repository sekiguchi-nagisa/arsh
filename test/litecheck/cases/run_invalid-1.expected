invalid RUN directive: `echo ${34 / "hello"}', caused by
(loaded):1:13: [semantic error] require `Int' type, but is `String' type
echo ${34 / "hello"}
            ^~~~~~~

